packages/vue/src/components/SpCrudUpdate/SpCrudUpdate.vue

```diff
@@ -45,9 +55,37 @@ export default defineComponent({
       type: String,
       required: true
     },
+
+    itemName: {
+      type: String,
+      required: true
+    },
+
+    itemData: {
+      type: Object,
+      required: true
+    },
   },
 
-  setup() {
+  setup(props, { emit }) {
+    // store
+    let $s = useStore()
+    let formData = reactive({ ...props.itemData })
+
+    // computed
+    let itemFields = computed(() => $s.getters[props.storeName + '/getTypeStructure'](props.itemName))
+    let creator = $s.getters['common/wallet/address']
+
+    let editItem = async () => {
+      await $s.dispatch(props.storeName + '/sendMsgCreatePost', { value: { ...formData, creator, id: props.itemData.id } })
```

Comment:
current [code](https://github.com/tendermint/vue/blob/8592c94e43592b17c7367606ace161d80a9fff74/packages/vue/src/components/SpTypeForm/SpTypeForm.vue#L230) is dynamic

Reply:
Not sure what you mean, sorry.